internal/pkg/agent/install/install_windows.go

```diff
@@ -25,6 +26,12 @@ func postInstall(topPath string) error {
 		return err
 	}
 
+	err = os.Remove(filepath.Join(topPath, version.PackageVersionFileName))
```

Comment:
We add the binary twice on windows because .zip does not support links.
To keep things consistent we add the package version side by side with all binaries but then post install we transform the binary in the top path in a link so we don't need the package version anymore

Reply:
comment, we will forget